lib/jekyll-import/importers/blogger.rb

```diff
@@ -247,7 +247,7 @@ def get_post_data_from_in_entry_elem_info
               else
                 filename = "%s-%s" %
                   [timestamp,
-                   CGI.escape(name.downcase).tr('+','-')]
+                   name.downcase.tr('+/\\:\'"<>{}?%*|.','-')]
```

Comment:
We might still need the CGI.escape here. I might also put this into its own method.

```ruby
def sanitized_filename(timestamp, name)
  return timestamp if name.nil?

  "%s-%s" % [
    timestamp,
    CGI.escape(name.downcase.tr('+/\\:\'"<>{}?%*|.', '-'))
  ]
end
```

Then remove from `if name.nil?` to the first `end`, and replace it with `filename = sanitized_filename(name, timestamp)` 😄